ide-support/revsblibrary.livecodescript

```diff
@@ -1719,3 +1719,80 @@ command revSBResolveIncludedExternal pPlatform, @xFile
       end if
    end if
 end revSBResolveIncludedExternal
+
+private command addToStringList pAppend, pValue, @xList
+   if pValue is among the lines of xList then
+      exit addToStringList
+   end if
+   
+   if xList is empty then
+      put pValue into xList
+   else if pAppend then
+      put return & pValue after xList
+   else
+      put pValue & return before xList
+   end if
+end addToStringList
+
+private command appendToStringList pValue, @xList
+   addToStringList true, pValue, xList
+end appendToStringList
+
+private command prependToStringList pValue, @xList
+   addToStringList false, pValue, xList
+end prependToStringList
+
+command revSBUpdateDeployParams pStack, pStandaloneSettings, @xDeployParams
+   local tModules, tAuxiliaryStackfiles, tStartupScript
+   
+   -- auxiliary_stackfiles key might not be empty
+   put pStandaloneSettings["auxiliary_stackfiles"] into tAuxiliaryStackfiles
+   
+   # AL-2015-04-12: [[ Standalone Extensions ]] Add chosen extensions to the standalone inclusions
+   # AL-2015-07-21: [[ Standalone Extensions ]] Automatically include dependecies, and load in order
+   repeat for each line tExtension in revIDEExtensionsOrderByDependency(pStandaloneSettings["extensions"])
+      local tExtensionFile
+      put revIDEExtensionModuleFile(tExtension) into tExtensionFile
+      appendToStringList tExtensionFile, tModules
+   end repeat
+   
+   repeat for each line tLibrary in pStandaloneSettings["scriptLibraries"]
+      # AL-2015-03-05: [[ Bug 14813 ]] Transform script library display name to stack name at deploy time
+      local tLibraryStack
+      put revSBStackNameFromScriptLibraryName(tLibrary) into tLibraryStack
+      if there is a stack tLibraryStack then
+         appendToStringList the effective filename of stack tLibraryStack, tAuxiliaryStackfiles
+         appendToStringList "revInternal__LoadLibrary" && tLibraryStack, tStartupScript
+      end if
+   end repeat
+   
+   -- If the datagrid is used, then include the library as an aux stack
+   if "data grid templates" is in the subStacks of stack pStack and there is a stack "revDataGridLibrary" then
+      appendToStringList the effective filename of stack "revDataGridLibrary", tAuxiliaryStackFiles
+   end if
```

Comment:
Is there any possibility that the new inclusions abstraction could be used to remove this special case in the future (not in this PR)?